pkg/apis/provisioning/v1alpha2/provisioner_validation.go

```diff
@@ -95,15 +101,11 @@ func (c *Cluster) validate() (errs *apis.FieldError) {
 	if c == nil {
 		return errs.Also(apis.ErrMissingField())
 	}
-	if len(c.Name) == 0 {
-		errs = errs.Also(apis.ErrMissingField("name"))
-	}
 	if len(c.Endpoint) == 0 {
+		// Endpoint cannot be resolved from in-cluster configuration and
```

Comment:
Consider skipping this extraneous comment. I think it's well covered in provisioner.go